README.md

```diff
@@ -49,6 +49,19 @@ COPY your-application.war ROOT.war
 The Tomcat instance can be configured through the environment variable `TOMCAT_PROPERTIES` which is
 a comma-separated list of `name=value` pairs appended to `catalina.properties`.
 
+### Stackdriver Logging
+When the Tomcat runtime is running on Google App Engine, all output to stdout/stderr is captured by Stackdriver 
```

Comment:
"Google App Engine" -> "Google App Engine flexible environment"
"captured" -> "forwarded to Stackdriver Logging and available in the Cloud Console Log Viewer"